test/jdk/sun/security/mscapi/EncodingMutability.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * @test
+ * @bug 8308808
+ * @requires os.family == "windows"
+ * @modules jdk.crypto.mscapi/sun.security.mscapi
```

Comment:
I think you just need `jdk.crypto.mscapi` on this line as you are not accessing the internal `sun.security.mscapi` classes.